Soujanya2004/wanderlust-2024
# Add Custom 404 Page Not Found

Fixes #13

This PR introduces a custom 404 page to improve user experience when navigating to non-existent routes.
- Updates routing to properly render the 404 page when no matching route is found.

### Changes
- Added styling to ensure the 404 page matches the overall design of the website.
- Implemented a "Back to Home" button to guide users back to the main page.

### Testing Instructions
1. Pull this branch.
2. Run npm install to install dependencies.
3. Navigate to a non-existent route, e.g., /non-existent-page.
4. Verify that the custom 404 page is displayed correctly.
5. Check the "Back to Home" button to ensure it works as expected and redirects to the homepage.

### Screenshots (if applicable)
![localhost_8000_listingl](https://github.com/user-attachments/assets/1aca1848-4f99-4108-9f63-b1bee4abb2e4)


### Checklist
